template/src/send_sample_request.js

```diff
@@ -140,10 +140,12 @@ function sendSampleRequest (group, name, version, method) {
     }
     requestParams.data = formData;
     requestParams.processData = false;
-    // GET and DELETE methods do not need content-type
-    if (method.toLowerCase() === 'get' || method.toLowerCase() === 'delete') {
-      delete requestParams.headers['Content-Type'];
-    }
+    // With no content-type header, browser will know it need to generate a proper content type for
+    // the form data when sending it. Fix #1122
```

Comment:
but how does the browser knows which content-type to use? Did you test this on multiple browsers?

Reply:
I tested on Chrome, and have tested on more browsers:
* firefox: multipart/form-data; boundary=---------------------------351499510217054383143954532765
* chrome: multipart/form-data; boundary=----WebKitFormBoundary98mmbfc7QnxEFKEe
* safari: multipart/form-data; boundary=----WebKitFormBoundaryVmoHCXIrZUuYVqry

I could not find a spec say how browser resolve the content-type, but according to jQuery document, it says:
> For cross-domain requests, setting the content type to anything other than application/x-www-form-urlencoded, multipart/form-data, or text/plain will trigger the browser to send a preflight OPTIONS request to the server. https://api.jquery.com/jquery.ajax/